admin/client/src/styles/legacy/_style.scss

```diff
@@ -1613,28 +1613,37 @@ form.member-profile-form {
 	}
 }
 
-.cms .cms-panel-padded {
-  &.ReportAdmin {
-    padding: $grid-gutter-width-half;
-    width: 100% !important;
+// Custom styles for ReportAdmin
+.ReportAdmin {
 
-    > .message {
-      margin-bottom: $grid-gutter-width-half;
-    }
+  > fieldset {
+    margin-top: -$grid-gutter-width-half;
 
-    > fieldset {
-      margin-top: -$grid-gutter-width-half;
+    // If form fields are at the top (typically .grid-field) then add additional space
+    > .form-group:first-child {
+      margin-top: $grid-gutter-width-half;
     }
+  }
 
-    .ss-gridfield-buttonrow {
-      margin-left: -20px;
-      margin-right: -20px;
-    }
+  > .message {
+    margin-bottom: $grid-gutter-width-half;
   }
 
-	.cms-content-view {
-		padding: 0 $grid-gutter-width-half $grid-y*1.5;
-	}
+  .toolbar--content {
+    margin-left: -$grid-gutter-width-half;
+    margin-right: -$grid-gutter-width-half;
+  }
+
+  &.cms-panel-padded {
```

Comment:
Talked to Paul about this, see comment on https://github.com/silverstripe/silverstripe-cms/issues/1491#issuecomment-245161859